nethunter-installer/build.py

```diff
@@ -145,7 +145,7 @@ def allapps(forcedown):
 		'Hackerskeyboard':
 			['https://f-droid.org/repo/org.pocketworkstation.pckeyboard_1039003.apk', '8c861c7540e6eeb006070d0f2d80134e75637066591d705b987e164c3fe87521ed694ac844c945eb74449aff8723ff039f793b2e3743aac73865f74bb248edf5'],
 		'Drivedroid':
-			['https://softwarebakery.com/apps/drivedroid/files/drivedroid-free-0.10.39.apk', 'f9db206642e783c478b9ba7a0fbee8a32b78cc6f919eb7a4fdf5e20dcc473e138917d043d3d17b15bd4e6b1fd202f422524085305b3658bd95903a346e8d1abc'],
+			['https://softwarebakery.com/apps/drivedroid/files/drivedroid-free-0.10.45.apk', 'ddf4e8B0fe85bd752fe1c726d277b0d18eb3197008d1060f1f4d9bfe1e7c71f7bdfd89d13817ea1a9195f8a11e354d6177fc358b1906ea7C675f2c6f6b0ef406'],
```

Comment:
Hi,

I'm not affiliated with the Kali NetHunter project but I'll share what I've seen so far.

There are uppercase characters, a 'B' and a 'C', where lowercase characters are expected, in the provided SHA512 hash which causes mismatch.
The rest worked merrily.